File: CollectionAPI/Models/Sell.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;
namespace CollectionAPI.Models;
public class Sell{

    [Column("id")]
    public int Id{get;set;}
    
    [Column("collectionid")]
    public int CollectionId{get;set;}

       [Column("merchantid")]
    public int MerchantId{get;set;}

       [Column("vehicleid")]
    public int VehicleId{get;set;}

       [Column("quantity")]
    public int Quantity{get;set;}

       [Column("netweight")]
    public  double NetWeight{get;set;}

       [Column("rateperkg")]
    public double RatePerKg{get;set;}

       [Column("date")]
    public DateTime Date{get;set;}
    public Sell(){
        Date=DateTime.Now;
    }
}

File: UI/FrontEnd/src/app/vendors/sell.ts
export class Sell {
    constructor(
                public id:number,
                public collectionId:number,
                public merchantId:number,
                public vehicleId:number,
                public quantity:number,
                public netWeight:number,
                public ratePerKg:number,
                public date:string){}
}
